### PR TITLE
Make Main function use CLI Parser.

### PR DESCRIPTION
**Pull request by beeperdeeper089**
Resolves: #
Authors:
@beeperdeeper089 

## Summary of changes
- Actually uses the CLI Parser.
- None of the arguments are actually used, next PR solves that.
  
## Caveats
Does this introduce any new bugs?
- None

## On approval
Merge.

### CODE REVIEW

**Comment by tobyplowy** [comment]
Build fails for me with 15 erroroonies.
[message.txt](https://github.com/GentenStudios/Phoenix/files/5759608/message.txt)

File: Phoenix/Client/Include/Client/Renderer/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,9 @@
+set(currentDir ${CMAKE_CURRENT_LIST_DIR})
+set(Headers
+        ${Headers}
+
+	${currentDir}/Window.hpp
+	${currentDir}/InputState.hpp
```
Comment (apachano) [inline Phoenix/Client/Include/Client/Renderer/CMakeLists.txt:6]: Is Input a part of Rendering? Should all client-side input handling be together?

File: Phoenix/Client/Source/Main.cpp [hunk 2]
```diff
@@ -27,12 +27,29 @@
 // POSSIBILITY OF SUCH DAMAGE.
 
 #include <Client/Client.hpp>
+#include <Common/CLIParser.hpp>
+#include <Common/Logger.hpp>
+
+#include <Common/CoreIntrinsics.hpp>
+#include <Common/Settings.hpp>
```
Comment (apachano) [inline Phoenix/Client/Source/Main.cpp:34]: Where are we using these?

**Review by apachano — COMMENTED** [review]
So why is there a second Window class? now we have phx::gfx::window and phx::render::window

Also whats the goal with render/ ? - rendering is definitely not my first though in terms of input (I know mouse interacts with the window, but the keeb doesn't. Maybe we create a controls or input dir instead?)

File: Phoenix/Client/Include/Client/Renderer/CMakeLists.txt (hunk 1 above)
Comment (beeperdeeper089) [reply Phoenix/Client/Include/Client/Renderer/CMakeLists.txt:6]: This was a prototype system i dont even know how it ended up on this branch

**Comment by tobyplowy** [comment]
Build fails. 
[message (2).txt](https://github.com/GentenStudios/Phoenix/files/5826421/message.2.txt)

**Comment by tobyplowy** [comment]
Builds and runs.

**Review by apachano — APPROVED** [review]
LGTM

**Pull request merged**